languages/elixir/exercises/concept/bitstrings/.docs/introduction.md

```diff
@@ -0,0 +1,44 @@
+Working with binary data is an important concept in any language, and Elixir provides an elegant syntax to write, match, and construct binary data.
+
+## Bitstrings
+
+In Elixir, binary data is referred to as the bitstring type. The binary datatype is a specific form of a bitstring, which we will discuss in a later exercise.
```

Comment:
I had to read this sentence a few times to understand it. "binary data" and "binary datatype" sound like the same thing if you're not paying extra attention. Maybe this will make it clearer?

```suggestion
In Elixir, binary data is referred to as the bitstring type. The binary data*type* (not to be confused with binary data in general) is a specific form of a bitstring, which we will discuss in a later exercise.
```

Reply:
Wonderful!